spec/compiler/parser/to_s_spec.cr

```diff
@@ -303,4 +303,20 @@ describe "ASTNode#to_s" do
       end
     end
     CRYSTAL
+
+  expect_to_s <<-'CR', <<-'CR'
+  {% verbatim do %}
+    {%
+      if !foo.empty? || 2 == 2
+        raise "oh noes"
+      end
+    %}
+  {% end %}
+  CR
+  {% verbatim do %}
+    {% if !foo.empty? || (2 == 2)
+    raise("oh noes")
+  end %}
+  {% end %}
+  CR
```

Comment:
**suggestion:** I'm sure the example could be reduced to a minimum? Something like `!a.b` should do I presume. It would fit great with the other specs for `!` in line 151f.